### DISCUSSION TITLE
Circular-Progress usage of `start-at`?

### DISCUSSION THREAD

**Discussion by TornaxO7** [elkowar/eww#570] [Q&A] [answered]
Hello! According to the docs of [`circular-progress`](https://elkowar.github.io/eww/widgets.html) I can use `start-at` for the angle where the circle should start. What angle is meant? `sin(angle)`? I expected to see the start at the top of the circle if I set `start-at = 90.0` (due to `sin(90.0) = 1`) but that wasn't the case. What am I understand wrong?

**Comment by VuiMuich** [comment]
I haven't done anything with circular-progress yet, but intuitively I would expect 0ˇ to be at 12 o'clock.

**Reply by TornaxO7** [reply]
But what would `90` mean? I couldn't figure it out yet :thinking:

**Reply by VuiMuich** [reply]
3 o'clock assuming `:clockwise` defaults to `true`

**Comment by anasrar** [comment, marked as answer]
`start-at` using percent unit not radian, and start on 3 o'clock

**Reply by TornaxO7** [reply]
Oooooh ok, thank you!
